README.md

```diff
@@ -1,127 +1,126 @@
 Docker open-oni
 ===============
 
-Run open-oni (community chronicling america fork) in Docker.
+(Production instructions TBD - for now this is just for doing open-oni dev)
 
-MySQL
------
+open-oni development
+--------------------
 
-```bash
-$ docker run -d \
-$   -p 3306:3306 \
-$   --name mysql \
-$   -e MYSQL_ROOT_PASSWORD=123456 \
-$   -e MYSQL_DATABASE=openoni \
-$   -e MYSQL_USER=openoni \
-$   -e MYSQL_PASSWORD=openoni \
-$   mysql
-```
+First clone open-oni: `git clone git@github.com:open-oni/open-oni.git`.
```

Comment:
This may be obvious, but I would recommend saying "clone from within docker-open-oni repository" or something to clarify where the open-oni repo goes.